ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Solaris.yml

```diff
@@ -102,7 +102,9 @@
     - cups
 
 - name: Extract cups 1.4.4
-  shell: tar -xf cups-1.4.4-source.tar.gz
+  shell: |
+    gzip -d cups-1.4.4-source.tar.gz
```

Comment:
I'd go with `gzip -cd cups-1.4.4-source.tar.gz | tar xpf -` as a one-liner and doesn't waste time creating an intermediate file